### PR TITLE
K arkadiusz/docs update

### PR DESCRIPTION
**Pull request by KArkadiusz**

### CODE REVIEW

**Comment by nalajcie** [comment]
small nitpick: the commit message is not completely valid - we're keeping our docs in Markdown, just using native MD syntax for embedding images (reStructuredText is a completely different format and image embedding is done there differently).

File: libc/functions/p/printf.part-impl.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# Synopsis
+# printf
```
Comment (damianloew) [inline libc/functions/p/printf.part-impl.md:1]: File to remove. Please look for similar ones.

**Review by damianloew — CHANGES_REQUESTED** [review]

**Review by damianloew — COMMENTED** [review]
`doc: adapt headers of libc functions to MD format` - it's not adapting to md format - it's just correcting the header in those files.

**Review by damianloew — APPROVED** [review]
Changed only commit messages in the last force push, LGTM

**Pull request merged**